README.md

```diff
@@ -23,6 +23,62 @@ This repo includes the following components:
 - [Apache Spark](http://spark.apache.org/) Connector: An Apache Spark connector that implements the Delta Sharing Protocol to read shared tables from a Delta Sharing Server. The tables can then be accessed in SQL, Python, Java, Scala, or R.
 - Delta Sharing Server: A reference implementation server for the Delta Sharing Protocol for development purposes. Users can deploy this server to share existing tables in Delta Lake and Apache Parquet format on modern cloud storage systems.
 
+## Feature List
```

Comment:
Or what's the best place, I feel it's better to summarize the key improvements in a table rather than let the users go through the release notes one by one.

Reply:
Should we create a new file RELEASE_NOTES.md (or something similar)?

Reply:
Seems based on sourcegraph.com, RELEASE_NOTES.md is commonly used for this. +1

Reply:
@zsxwing Do you have any suggestions? Thanks!

Reply:
added RELEASE_NOTES.md.